client.go

```diff
@@ -265,6 +266,12 @@ func setupChannel(remotePort int, sshPort int, login string) int {
 	return localPort
 }
 
+func (c *SingleServerClient) CleanupDeploy(deployIdForCleanup string) error {
+	req := &SetActiveByIdRequest{deployIdForCleanup}
+	var reply SetActiveByIdReply
```

Comment:
It seems weird that you are re-using this structure. The name itself is confusing. I'd suggest making 2 new types (CleanupRequest and CleanupReply) and using those instead.

If later, we find there is a lot of commonality between the 2 requests/response types, we can think of a better name and unify them (However, I suspect at the top level, we'll always want seperate types for flexibility)